qcodes/instrument_drivers/Galil/dmc_41x3.py

```diff
@@ -0,0 +1,964 @@
+                map(
+                    lambda i, j: i - j,
+                    self.left_top_position,
+                    self.left_bottom_position,
+                )
+            )
+        )
+        self.norm_b = np.linalg.norm(b)
+        self._b = b / self.norm_b
+
+        c = np.array(
+            tuple(
+                map(lambda i, j: i - j,
+                    self.right_top_position,
+                    self.left_top_position)
+            )
+        )
+        self.norm_c = np.linalg.norm(c)
+        self._c = c / self.norm_c
+
+        n = np.cross(self._a, self._b)
+        norm_n = np.linalg.norm(n)
+        self._n = n / norm_n
+
+        intercept = np.array(sum(-1 * self._n * self.left_bottom_position))
+        self._plane_eqn = np.append(self._n, intercept)
+
+    def move_motor_a_by(self, distance: float) -> None:
+        """
+        Moves motor A by distance given in micro meters
+        """
+        a = self.controller.motor_a
+
+        d = _convert_micro_meter_to_quadrature_counts(distance)
+
+        a.relative_position(d)
+        a.speed(self.speed)
+        a.acceleration(self.acceleration)
+        a.deceleration(self.deceleration)
+        a.servo_here()
+        a.begin()
+        a.wait_till_motor_motion_complete()
+
+    def move_motor_b_by(self, distance: float) -> None:
+        """
+        Moves motor B by distance given in micro meters
+        """
+        b = self.controller.motor_b
+
+        d = _convert_micro_meter_to_quadrature_counts(distance)
+
+        b.relative_position(d)
+        b.speed(self.speed)
+        b.acceleration(self.acceleration)
+        b.deceleration(self.deceleration)
+        b.servo_here()
+        b.begin()
+        b.wait_till_motor_motion_complete()
+
+    def move_motor_c_by(self, distance: float) -> None:
+        """
+        Moves motor B by distance given in micro meters
+        """
+        c = self.controller.motor_c
+
+        d = _convert_micro_meter_to_quadrature_counts(distance)
+
+        c.relative_position(d)
+        c.speed(self.speed)
+        c.acceleration(self.acceleration)
+        c.deceleration(self.deceleration)
+        c.servo_here()
+        c.begin()
+        c.wait_till_motor_motion_complete()
+
+    def _setup_motion(self,
+                      rel_vec: np.ndarray,
+                      d: float,
+                      speed: float) -> None:
+
+        pos = self.controller.absolute_position()
+
+        a = int(np.floor(rel_vec[0] * d))
+        b = int(np.floor(rel_vec[1] * d))
+        c = int(np.floor(rel_vec[2] * d))
+
+        self._target = np.array([pos["A"] + a, pos["B"] + b, pos["C"] + c, 1])
+
+        if np.dot(self._plane_eqn, self._target) < 0:
+            raise RuntimeError(
+                f"Cannot move to {self._target[:3]}. Target location is below "
+                f"chip plane."
+            )
+
+        sp_a = int(np.floor(abs(rel_vec[0]) * speed))
+        if sp_a % 2 != 0:
+            sp_a += 1
```

Comment:
is it better to have higher or lower speeds? :) i guess "1" is such a small difference that either is fine. and the speed limit is so high that noone would even want to use that, right?
```suggestion
            sp_a -= 1
```